fetch/client/src/main/scala/endpoints4s/fetch/ReadableStream.scala

```diff
@@ -0,0 +1,253 @@
+package endpoints4s.fetch
+
+import org.scalajs.dom.ReadableStreamReader
+import org.scalajs.dom.WriteableStream
+
+import scala.annotation.nowarn
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSGlobal
+
+//FIXME copypasta from scalajs-dom https://github.com/scala-js/scala-js-dom/pull/628
```

Comment:
Should we use the same package, then? `org.scalajs.dom`?

Reply:
We do use `org.scalajs.dom.ReadableStream[A]` for `Chunks[A]`, so to use this package for internal classes we would need to rename them too, right? What do we gain here then?

Reply:
Hm, I am not sure then, but since the definitions below are all private that’s not important anyway :)